server/channels/app/imaging/orientation.go

```diff
@@ -37,19 +37,19 @@ const (
 func MakeImageUpright(img image.Image, orientation int) image.Image {
 	switch orientation {
 	case UprightMirrored:
-		return imaging.FlipH(img)
+		return transform.FlipH(img)
 	case UpsideDown:
-		return imaging.Rotate180(img)
+		return transform.Rotate(img, 180, &transform.RotationOptions{ResizeBounds: true})
 	case UpsideDownMirrored:
-		return imaging.FlipV(img)
+		return transform.FlipV(img)
 	case RotatedCWMirrored:
-		return imaging.Transpose(img)
+		return transform.Rotate(transform.FlipH(img), -90, &transform.RotationOptions{ResizeBounds: true})
 	case RotatedCCW:
-		return imaging.Rotate270(img)
+		return transform.Rotate(img, 90, &transform.RotationOptions{ResizeBounds: true})
 	case RotatedCCWMirrored:
-		return imaging.Transverse(img)
+		return transform.Rotate(transform.FlipV(img), -90, &transform.RotationOptions{ResizeBounds: true})
 	case RotatedCW:
-		return imaging.Rotate90(img)
+		return transform.Rotate(img, 270, &transform.RotationOptions{ResizeBounds: true})
```

Comment:
I am a bit unclear as to why we need to use 270 here when the direction of rotation is specified as clockwise in the documentation. 

https://pkg.go.dev/github.com/anthonynsimon/bild@v0.14.0/transform#Rotate

> Parameters angle is in degrees and it's applied clockwise.